api/v1alpha08/sonataflow_types.go

```diff
@@ -657,8 +657,7 @@ type SonataFlowSpec struct {
 	// PodTemplate describes the deployment details of this SonataFlow instance.
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="podTemplate"
 	PodTemplate PodTemplateSpec `json:"podTemplate,omitempty"`
-	// Persists service to a datasource of choice. Ephemeral by default.
-	// +optional
+	// Persistence defines the database persistence configuration for the workflow
 	Persistence *PersistenceOptions `json:"persistence,omitempty"`
```

Comment:
Now that we are going to share the PersistenceOptions type between services and workflows, and the other Postgresql types too, maybe it could make sense to have these structs in a separate file, maybe sonataflow_persitence_types.go
@ricardozanini @jordigilh  wdyt?

Reply:
I think `PersistenceOptionsSpec` makes more sense because this is used in both the platform and workflow objects.

Reply:
Done.